pkg/minikube/minikube.go

```diff
@@ -43,6 +43,7 @@ func SetUp(name string) error {
 		fmt.Print("\n")
 		fmt.Println("To finish setting up networking for minikube, run the following command in a separate terminal window:")
 		fmt.Println("    minikube tunnel --profile knative")
+		fmt.Println("The tunnel window must remain open until you are done with the quickstart environment.")
```

Comment:
```suggestion
		fmt.Println("The tunnel command must be running in a terminal window any time when using the knative quickstart environment.")
```

Reply:
the message should let the user know any time using the environment in later time they need to have the tunnel command running